fr/development/routing.rst

```diff
@@ -1517,11 +1516,12 @@ css, javascript, images et autres fichiers statiques de votre application::
     // Génère une URL pointant vers APP/webroot/files/upload/photo.png
     $file = Asset::url('files/upload/photo.png');
 
-Les méthodes ci-dessus acceptent également un tableau d'options comme deuxième paramètre:
+Les méthodes ci-dessus acceptent également un tableau d'options comme deuxième paramètre::
```

Comment:
This should end with only one :. Ending a paragraph with `::` means the next block is a code block which isn't the case here.

Reply:
Fixed in 12a4afb